pkg/rhttp/datatx/manager/spaces/spaces.go

```diff
@@ -84,18 +84,13 @@ func (m *manager) Handler(fs storage.FS) (http.Handler, error) {
 
 			// TODO refactor: pass Reference to Upload & GetOrHeadFile
```

Comment:
I think this TODO should live on the other side, on `fs.Upload`. But nothing for this PR.

Reply:
I didn't understand the comment anyways so I removed it in https://github.com/cs3org/reva/pull/2383